index.js

```diff
@@ -11,6 +11,7 @@ const colors = {
   blue: '#7FDBFF',
   lightGray: '#DDDDDD',
   gray: '#AAAAAA',
+  orange: '#FF851B',
```

Comment:
haha well that looks good, it could have been the lack of sleep. Let's make it yellow then :+1: